setup.py

```diff
@@ -50,7 +50,7 @@ def get_packages(package):
     packages=get_packages(PACKAGE),
     package_data={PACKAGE: ["py.typed"]},
     data_files=[("", ["LICENSE.md"])],
-    install_requires=["databases>=0.2.1", "typesystem"],
+    install_requires=["anyio>=3.0.0,<4", "databases>=0.2.1", "typesystem"],
     extras_require={
         "postgresql": ["asyncpg"],
```

Comment:
```suggestion
        "tests": ["anyio>=3.3.0,<4"],
        "postgresql": ["asyncpg"],
```

Reply:
sorry this suggestion is wrong test deps go here

https://github.com/encode/orm/blob/master/requirements.txt

Reply:
@graingert  we can add anyio to the `requirements.txt` instead of extra requirements.